### PR TITLE
port_bounce.py test enhancement

### PR DESCRIPTION
**Pull request by Naresh-ibm**
enahnaced the port_bounce.py test by adding the multipath verification.
removed the port_number dependency passing it through yaml.

Signed-off-by: Naresh Bannoth <nbannoth@in.ibm.com>

### CODE REVIEW

**Comment by Naresh-ibm** [comment]
linux-7sxk:~ # avocado run port_bounce_latest.py -m port_bounce.py.data/port_bounce.yaml
JOB ID     : 48ff7a2f73f6f5e64d217b578eeaf39c5da88985
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2019-11-15T01.46-48ff7a2/job.log
 (1/1) port_bounce_latest.py:PortBounceTest.test;run-6b9e: PASS (1015.20 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 1015.30 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2019-11-15T01.46-48ff7a2/results.html
linux-7sxk:~ #

**Comment by Naresh-ibm** [comment]
[port_bounce_latest_without_port_name.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/3851968/port_bounce_latest_without_port_name.txt)

File: io/disk/port_bounce.py [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+            test_ports.append(port)
+            for _ in range(self.count):
+                self.porttoggle(test_ports, self.sbt)
+
+        self.log.info("Long port bounce for individual ports:")
+        for port in self.port_ids:
+            test_ports = []
+            test_ports.append(port)
+            for _ in range(self.count):
+                self.porttoggle(test_ports, self.lbt)
```
Comment (narasimhan-v) [inline io/disk/port_bounce.py]: `test_ports` is initialised multiple times, take care of that.
Also, `test_ports` can be found once, and used the subsequent times.
And the 2 sequences do the same thing except for the port toggle time, so they can be made into a function and called twice.

File: io/disk/port_bounce.py [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+        for port in test_ports:
+            self.log.info("verify port %s disable in %s", port, test_ports)
+            port_string = ".*%s.*Disabled" % port
+            obj = re.search(port_string, switch_info)
+            if obj:
+                self.log.info("Port %s is disabled", port)
+            else:
+                self.log.debug("switch_info: %s", switch_info)
+                msg = "Port %s is failed to disable" % port
+                self.failure_list[port] = msg
```
Comment (narasimhan-v) [inline io/disk/port_bounce.py]: `verify_port_disable()` and `verify_port_enable()` are same functions, with just different checks. They can be made into one function with just the check different (similar to `verify_port_toggle_host()`)

**Comment by Naresh-ibm** [comment]
closing this in favour of #1424

**Pull request closed without merging**